bin/helpers.rb

```diff
@@ -62,3 +62,12 @@ def validate_ha_label!(label, script_name)
 def valid_token(token)
   token.length == 40
 end
+
+def mobile_resources_commit()
+  _output, error, status = Open3.capture3("cat", "./bin/mobile_resources_commit")
```

Comment:
Maybe there's a better way to read the content of the file.
I used `cat` just because I'm not familiar with Ruby

Reply:
I think it's perfectly fine, maybe the location of it since `bin` symbols executables, but definitely not a big deal.

Reply:
Thanks!
I'm using the `File` module now